File: dotnet/DesignPatterns/CreationalDesignPatterns/AbstractFactoryDesignPattern/AbstractFactoryDesignPattern.Example1/WindowsMenu.cs
namespace AbstractFactoryDesignPattern.Example1;

public class WindowsMenu : Menu
{
	public override void Render()
	{
		Console.WriteLine("Rendering Windows Menu");
	}
	public override void AddItem(string item)
	{
		Console.WriteLine($"Adding '{item}' to Windows Menu");
	}
}

File: java/DesignPatterns/CreationalDesignPatterns/AbstractFactoryDesignPattern/AbstractFactoryDesignPattern.Example1/src/main/java/creational/abstractfactory/example1/models/concretes/WindowsMenu.java
package creational.abstractfactory.example1.models.concretes;

import creational.abstractfactory.example1.models.abstracts.Menu;

public class WindowsMenu extends Menu {
    public void render()           { System.out.println("Rendering Windows Menu"); }
    public void addItem(String item) { System.out.println("Adding '" + item + "' to Windows Menu"); }
}
